### ISSUE TITLE
Create the Header(Navbar) and the Footer section for all the pages.

### ISSUE THREAD

**Issue by Aryan-Kumar-Thakur** [gdsc-jssstu/Programming_Community#2]
You have to create header and footer section as shown in UI/UX design.
 The header contains the nabar contain all sections/pages with the links to visit that page. It should contain proper animation to show that currently we are on what page and also with proper hover effect.
The footer contains the details about social media handles  and the contact information through which the contact with programming community.
Again you use dummy links whenever needed.

Request to assign the issue to avoid any conflict.

**Comment by Anshikaashuklaa** [comment]
Hi @Aryan-Kumar-Thakur Can you please assign this also to me?

**Comment by Aryan-Kumar-Thakur** [comment]
> Hi @Aryan-Kumar-Thakur Can you please assign this also to me?

Yeah sure, I assign this to you. Once you done with your work feel free to genereate a pull request. All the best 👍.

**Comment by Anshikaashuklaa** [comment]
Thankyou :)

**Comment by M27afk** [comment]
Any updates @Anshikaashuklaa ?
